python/py-aiohttp/Portfile

```diff
@@ -0,0 +1,41 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+
+name                py-aiohttp
+version             3.3.2
+categories-append   devel
+platforms           darwin
+license             Apache-2
+
+python.versions     35 36
```

Comment:
I see no reason to add a py35 version. The latest version of python 3 is 3.6, so only add that.

Reply:
Someone forced to use Python 3.5 for whatever reason may need to make use the library.

Reply:
You could say the same for python 3.4, 3.3 etc. We are currently trying to remove python 3.4 and earlier modules and will extend that to 3.5 when it is no longer supported by upstream. I don't see a point in adding it unless some other port in Macports specifically needs it. However, since you are volunteering to maintain this port, then if you are ok with supporting the extra version, it is fine with me to leave it in.